spec/unit/application/client_spec.rb

```diff
@@ -275,9 +275,9 @@
     ARGV.replace(@original_argv)
   end
 
-  it "should set the colored output to false by default on windows and true otherwise" do
+  it "should set the colored output to true by default on windows and true on all other platforms as well" do
```

Comment:
I don't think we need a conditional here at all anymore. The appveyor run should adequately test this scenario.